app/src/org/commcare/dalvik/activities/CommCareHomeActivity.java

```diff
@@ -880,7 +874,7 @@ private void startNextFetch() {
     // extremely long method
 
     private void readyToProceed(final CommCareSession session) {
```

Comment:
Not sure what you mean. readyToProceed still uses the session argument. And what do you mean by both?